includes/templates/amp-paired-browsing.php

```diff
@@ -43,16 +43,9 @@
 						<span class="general">
 							<?php esc_html_e( 'The navigated URL is not available for paired browsing.', 'amp' ); ?>
 						</span>
-
-						<span class="invalid-amp">
-							<?php esc_html_e( 'The navigated page does not have an AMP counterpart due to invalid AMP markup being kept.', 'amp' ); ?>
-						</span>
 					</div>
 
 					<div class="dialog-buttons">
-						<button class="button exit">
-							<?php esc_html_e( 'Exit', 'amp' ); ?>
-						</button>
```

Comment:
This can be restored if we capture link clicks in the preview and send a message to the parent containing the `href` being navigated to.